.ci/travis/travis.sh

```diff
@@ -443,6 +449,19 @@ no-exception-test-alot-of-project1)
       --config checks-nonjavadoc-error.xml --checkstyleVersion $CS_POM_VERSION
   ;;
 
+
+no-error-pmd)
+  CS_POM_VERSION=$(mvn -e -q -Dexec.executable='echo' -Dexec.args='${project.version}' \
+                     --non-recursive org.codehaus.mojo:exec-maven-plugin:1.3.1:exec)
+  echo CS_version: ${CS_POM_VERSION}
+  mkdir -p .ci-temp/
+  cd .ci-temp/
+  git clone https://github.com/pmd/pmd.git
+  cd pmd
+  mvn -e install checkstyle:check -Dcheckstyle.version=${CS_POM_VERSION}
+  removeFolderWithProtectedFiles pmd
```

Comment:
Please do 
`cd ..`
Before this line, to remove folder from top, not from inside

Reply:
this is still not done.